### PR TITLE
Exposes ReactDOM to the window for the Plugins API.

### PR DESCRIPTION
**Pull request by gojutin**
This is my first pull request so hopefully I got this mostly right. This PR exposes ReactDOM to the window object for consumption by React-based plugins and will result in a ~120KB bundle size reduction in the React template.

### CODE REVIEW

File: packages/typescriptlang-org/src/templates/play.tsx [hunk 1]
```diff
@@ -51,85 +54,141 @@ const Play: React.FC<Props> = (props) => {
...
+              "main",
+              !!main,
+              "ts",
+              !!ts,
+              "sandbox",
+              !!sandbox,
+              "playground",
+              !!playground
+            )
+          }
```
Comment (orta) [inline packages/typescriptlang-org/src/templates/play.tsx]: ergh - from 9 readable lines of code to 30 

I'll add this file to the prettier ignore, and merge your changes manually if you accept the CLA

Comment (gojutin) [reply packages/typescriptlang-org/src/templates/play.tsx]: Oops... didn't think to check if prettier formatted it. If it's easier to do anything on my end, just let me know and I'll be happy to.

Comment (orta) [reply packages/typescriptlang-org/src/templates/play.tsx]: Nah, I'll merge your changes in manually now and deploy straight away - I already have a few changes sitting by

**Comment by orta** [comment]
OK, this is merged manually - sorry @gojutin for stealing your credit :D

**Pull request closed without merging**